en/advanced-topics/engine-customization.md

```diff
@@ -214,6 +214,10 @@ The `gulp` command will pack all the code in **bultin** into file `jsb-builtin.j
 
 After customizing the `jsb-adapter`, the editor will copy files in **dist** folder to the **jsb-adapter** folder in the project when you **build** the native platform in the **Build** panel.
 
+#### 3.2.3 Customization after Creator v2.0.7 (v2.0.7 included)
+
+After Creator v2.0.7, the complicated mannual compiling operation is deprecated. You can straightly edit the source code in both **builtin** and **engine** folder. Launch Creator after finishing editing, the source code will be automatically compiled when Creator launching.
```

Comment:
After Creator v2.0.7, Jsb-adapter abandoned the cumbersome manual compilation operation. You can directly modify the source code in directories **builtin** and **engine**. When the modification is complete, open the editor and the editor will automatically compile this part of the source code at startup.